reana_server/rest/info.py

```diff
@@ -45,34 +52,7 @@ def info(user, **kwargs):  # noqa
         200:
           description: >-
             Request succeeded. The response contains general info about the cluster.
-          schema:
-            type: object
-            properties:
-              workspaces_available:
-                type: object
-                properties:
-                  title:
-                    type: string
-                  value:
-                    type: array
-                    items:
-                      type: string
-              default_workspace:
-                type: object
-                properties:
-                  title:
-                    type: string
-                  value:
-                    type: string
-              compute_backends:
-                type: object
-                properties:
-                  title:
-                    type: string
-                  value:
-                    type: array
-                    items:
-                      type: string
+          schema: InfoSchema
```

Comment:
Is this change OK with respect to the OpenAPI specification and client generation and such? I think we use explicit YAML everywhere for the other endpoints?

Reply:
If we generate the same OpenAPI, and Bravado is happy, then it should be good then! 

P.S. We might want to harmonise this later elsewhere for the other endpoints, just for consistency's sake, but this will take time.